.github/workflows/clp-lint.yaml

```
@@ -33,6 +33,10 @@ jobs:
         shell: "bash"
         run: "npm install -g @go-task/cli"
 
+	  - name: "Install uv"
```

actionlint (1.7.7)

could not parse as YAML: yaml: line 36: found character that cannot start any token

(syntax-check)